pubky-client/bindings/js/src/api/auth.rs

```diff
@@ -16,6 +18,18 @@ use super::super::constructor::Client;
 
 use wasm_bindgen::prelude::*;
 
+/// Optional parameters for the `signup` method.
+#[derive(Tsify, Serialize, Deserialize, Debug, Default)]
+#[tsify(into_wasm_abi, from_wasm_abi)]
+#[serde(rename_all = "camelCase")]
+pub struct SignupOptions {
+    /// The signup token or invite code.
+    pub signup_token: Option<String>,
+
+    /// A boolean indicating acceptance of the Terms of Service.
+    pub accept_tos: Option<bool>,
+}
```

Comment:
This is the generated Typescript interface from your current code:

```ts
export interface SignupOptions {
    /**
     * The signup token or invite code.
     */
    signupToken: string | null;
    /**
     * A boolean indicating acceptance of the Terms of Service.
     */
    acceptTos: boolean | null;
}
```

As you see, it allows `null` but doesn't allow the fields to be `undefined`. You need to add `[tsify(optional)]` to allow them to be `undefined`. Otherwise, you always need to set each field to null explicitly instead of being able to omit them.


```suggestion
#[derive(Tsify, Serialize, Deserialize, Debug, Default)]
#[tsify(into_wasm_abi, from_wasm_abi)]
#[serde(rename_all = "camelCase")]
pub struct SignupOptions {
    /// The signup token or invite code.
    #[tsify(optional)]
    pub signup_token: Option<String>,

    /// A boolean indicating acceptance of the Terms of Service.
    #[tsify(optional)]
    pub accept_tos: Option<bool>,
}
```